packages/core/src/components/dropdown/dropdown.scss

```diff
@@ -131,7 +173,17 @@
     box-shadow: rgb(0 0 0 / 10%) 0 2px 3px 0;
     border-radius: 0 0 4px 4px;
     overflow-y: auto;
-    @include tds-scrollbar;
+    @include tds-dropdown-scrollbar;
+
+    // TODO: Why cant we put this block in the mixin tds-dropdown-scrollbar? Can not get that to work.
```

Comment:
Regarding this TODO, a mixin itself doesn't have any inherent logic capabilities to detect browser support. It blindly outputs the styles wherever it's included. So, if a mixin includes both -webkit- prefixed styles for scrollbar customization and @supports not rules intended for browsers that do not support those prefixes, it cannot dynamically adjust based on the client's browser.